DESCRIPTION

```diff
@@ -29,7 +29,8 @@ Imports:
     ggplot2,
     lifecycle,
     magrittr,
-    mmrm (>= 0.1.0),
+    mmrm (>= 0.2.2),
```

Comment:
just to confirm
is it `>= 0.2.2` (i.e. last released version is fine) or rather `> 0.2.2` (i.e. development version)?

Reply:
thx @pawelru, last released version is fine